libraries/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -606,26 +606,28 @@ export class WorkspaceInstallManager extends BaseInstallManager {
       const tempNodeModulesPath: string = `${subspace.getSubspaceTempFolder()}/node_modules`;
       const modulesFilePath: string = `${tempNodeModulesPath}/${RushConstants.pnpmModulesFilename}`;
 
-      const modulesContent: string = FileSystem.readFile(modulesFilePath);
-      const yamlContent: IPnpmModules = yaml.load(modulesContent, { filename: modulesFilePath });
-      const { hoistedDependencies } = yamlContent;
-      const subspaceProject: RushConfigurationProject = subspace.getProjects()[0];
-      const projectNodeModulesPath: string = `${subspaceProject.projectFolder}/node_modules`;
-      for (const value of Object.values(hoistedDependencies)) {
-        for (const [filePath, type] of Object.entries(value)) {
-          if (type === 'public') {
-            // If we don't already have a symlink for this package, create one
-            if (!fs.existsSync(`${projectNodeModulesPath}/${filePath}`)) {
-              // Ensure origin folder exists
-              const parentDirSep: string[] = `${projectNodeModulesPath}/${filePath}`.split(path.sep);
-              parentDirSep.pop();
-              const parentDir: string = parentDirSep.join(path.sep);
-              await FileSystem.ensureFolderAsync(parentDir);
-              BaseLinkManager._createSymlink({
-                linkTargetPath: `${tempNodeModulesPath}/${filePath}`,
-                newLinkPath: `${projectNodeModulesPath}/${filePath}`,
-                symlinkKind: SymlinkKind.Directory
-              });
+      if (await FileSystem.existsAsync(modulesFilePath)) {
+        const modulesContent: string = FileSystem.readFile(modulesFilePath);
+        const yamlContent: IPnpmModules = yaml.load(modulesContent, { filename: modulesFilePath });
+        const { hoistedDependencies } = yamlContent;
+        const subspaceProject: RushConfigurationProject = subspace.getProjects()[0];
```

Comment:
Is it guaranteed that every `Subspace` will always contain at least one project?  If so, then we should add that to the `/** */` docs for [Subspace.getProjects](https://github.com/william2958/rushstack/blob/816c1463d4f6c396caf4b923b35fa978bee9c9ab/libraries/rush-lib/src/api/Subspace.ts#L60) to ensure that this constraint isn't accidentally changed in the future.

Reply:
This is only guaranteed here, as we're verifying that this subspace only have 1 project on line 603